src/renderer/actions.ts

```diff
@@ -119,7 +119,52 @@ export function setGlobeMousePosition(position: GeographicPosition): Action {
     return {type: SET_GLOBE_MOUSE_POSITION, payload: {position}};
 }
 
-export function setGlobeViewPosition(position: GeographicPosition): Action {
+export function setGlobeViewPosition(position: GeographicPosition): ThunkAction {
+    return (dispatch: Dispatch, getState: GetState) => {
+        dispatch(setGlobeViewPositionImpl(position));
+        if (position) {
+            const baseDir = selectors.workspaceBaseDirSelector(getState());
+            assert.ok(baseDir);
+            const resource = selectors.selectedResourceSelector(getState());
+            const layer = selectors.selectedVariableImageLayerSelector(getState());
+            if (!layer || ! resource) {
+                return;
+            }
+
+            const indexCoords = findVariableIndexCoordinates([resource], layer);
```

Comment:
findVariableIndexCoordinates is inappropriate here. Impl. a new method, e.g. `getNonSpatialIndices` that outputs what is required here.